lib/config/presets/index.ts

```diff
@@ -110,7 +112,9 @@ export function parsePreset(input: string): ParsedPreset {
   let presetName: string;
   let tag: string | undefined;
   let params: string[] | undefined;
-  if (str.startsWith('github>')) {
+  if (str.startsWith('http://') || str.startsWith('https://')) {
+    presetSource = 'http';
+  } else if (str.startsWith('github>')) {
```

Comment:
any reason to have this first instead of adding it as later condition?

I think it should come after the other platforms.